app/assets/stylesheets/space_table.css

```diff
@@ -0,0 +1,141 @@
+.view-as-table,
```

Comment:
Everything in .view-as-table is related to showing a table in the search, as an alternative view. 

That is commented out for now. We should probably extract all that code and remove it from the project all-together, if we aren't going to use it.